test/test-najax.js

```diff
@@ -73,6 +73,14 @@ describe('url', function (next) {
   }
 
   it('should accept plain URL', function (done) {
+    // An extra nock seems to be being
+    // registered and affecting the subsequent tests
+    // (I beleive it is caused by:
+    // https://github.com/najaxjs/najax/commit/54cfac6aa0b7cf8a89efae0f39d1f054c8859de0
+    // commenting out the 'default to "GET"' test also fixes this)
+    // make an extra call to najax in order to clear it
+    najax({url: 'http://www.example.com'})
```

Comment:
Ah sorry about this, we should probably clear all the nocks after each test?

e.g.
```js

afterEach(function () {
   nock.cleanAll()
})
```